wazo_confd/plugins/user_import/creators.py

```diff
@@ -187,6 +188,14 @@ def create(self, fields, tenant_uuid):
         exten = fields.get('exten')
         context = fields.get('context')
         if exten and context:
+            firstname = fields.get('firstname')
```

Comment:
I can't figure out why you tested for the firstname here

Reply:
To avoid breaking too much the current implementation
If you want to import only a exten/context without user

Reply:
After investigation I cannot remove it. I tried to make code clearer